relay/network_graph/graph.py

```diff
@@ -296,35 +295,46 @@ def total_cost_from_start_to_dst(
 
 class CapacityAccumulator(alg.CostAccumulator):
     """This is being used to find a path with the maximum capacity"""
-    def __init__(self, *, timestamp, max_hops=None):
+    def __init__(self, *, timestamp, capacity_imbalance_fee_divisor, max_hops=None):
         if max_hops is None:
             max_hops = math.inf
         self.max_hops = max_hops
         self.timestamp = timestamp
+        self.capacity_imbalance_fee_divisor = capacity_imbalance_fee_divisor
 
-    def get_capacity(self, node, dst, edge_data):
-        balance = balance_with_interests(
+    def get_balance(self, node, dst, edge_data):
+        return balance_with_interests(
             get_balance(edge_data, node, dst),
             get_interest_rate(edge_data, node, dst),
             get_interest_rate(edge_data, dst, node),
             self.timestamp - get_mtime(edge_data))
-        return balance + get_creditline(edge_data, dst, node)
+
+    def get_capacity(self, node, dst, edge_data):
+        return self.get_balance(node, dst, edge_data) + get_creditline(edge_data, dst, node)
 
     def zero(self):
-        return (-math.inf, 0)  # We use (- capacity, num_hops) as cost
+        return (-math.inf, 0, 0)  # We use (- capacity, num_hops, last_hop_fee) as cost
 
     def total_cost_from_start_to_dst(
         self, cost_from_start_to_node, node, dst, edge_data
     ):
 
         capacity_from_start_to_node = - cost_from_start_to_node[0]
```

Comment:
Can we not use a namedtuple for the cost to make it more clear than cost[0], cost[1], cost [2].